packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -2179,27 +2179,16 @@ class _RenderFocusTrap extends RenderProxyBoxWithHitTestBehavior {
     return hitTarget;
   }
 
-  /// The focus dropping behavior is only present on desktop platforms
-  /// and mobile browsers.
+  /// The focus dropping behavior is only present on web (mobile and desktop).
   bool get _shouldIgnoreEvents {
-    switch (defaultTargetPlatform) {
-      case TargetPlatform.android:
-      case TargetPlatform.iOS:
-        return !kIsWeb;
-      case TargetPlatform.linux:
-      case TargetPlatform.macOS:
-      case TargetPlatform.windows:
-      case TargetPlatform.fuchsia:
-        return false;
-    }
+    return !kIsWeb;
   }
 
   @override
   void handleEvent(PointerEvent event, HitTestEntry entry) {
     assert(debugHandleEvent(event, entry));
     if (event is! PointerDownEvent
       || event.buttons != kPrimaryButton
-      || event.kind != PointerDeviceKind.mouse
```

Comment:
This is causing failures because it seems to intercept drag events.  I'm checking if this is still viable.

Reply:
This will now compete with scrolling drags - which will push gesture acceptance out. Actually reading this PR, I'm not sure that changing this condition is a good idea.